src/CSS/login.css

```diff
@@ -0,0 +1,134 @@
+.register-login-wrapper{
+    height: auto;
+    background: #D7EFEE!important;
+    margin: 0px auto;
+    margin-top: 13px;
+    top: 16%;
+    position: relative;
+    z-index: 2;
+    min-width: 320px;
+    padding-bottom: 60px;
+    padding-top: 30px;
+}
+.signup-content,.login-content{
+    top: 10%;
+    font-style: normal;
+    font-weight: bold;
+    line-height: normal;
+    font-size: 24px;
+    text-align: center;
+    margin: 0 auto;
+}
+.signup-content .signin-intro,.login-content .signin-intro {
+    margin-bottom: 20px
+}
+.signup-content span,.login-content span {
+    font-style: normal;
+    font-weight: normal;
+    line-height: 24px;
+    font-size: 14px;
+    text-align: center;
+}
+.login-content a div{
+    height: 37px;
+    background: #FFFFFF;
+    box-shadow: 1px 4px 21px rgba(0, 0, 0, 0.12);
+    border-radius: 5px;
+    margin-bottom: 10px;
+    color:#000;
+}
+.signup-content a span.create-account,
+.login-content a span.create-account{
+    background: none;
+    box-shadow: none;
+    text-align: right;
+    float: right;
+}
+.login-content .engine-img{
+    width: 100%;
+    overflow: hidden;
+    box-shadow: none;
+    padding-top: 1px;
+    text-align: left;
+    padding-left: 20px;
+}
+.login-content .engine-img img{
+    width: 25px;
+}
+.signup-content{
+    display: none
+}
+.alert {
+    font-size: 13px;
+    font-weight: normal;
+}
+@media (max-width:320px){
+    .bs-example-modal-sm,.register-login-wrapper{
+        padding-right: 0;
+        padding-left: 0;
+    }
+}
+
+/* write article layout */
+
+img{
+    width:100%;
+    height: auto;
+}
+.img-circle, .header{
+    width:60px;
+    height:60px;
+    overflow: hidden;
+    border-radius: 50%;
+}
+/* write article layout */
+.bordered{
+    margin: 18px 18px 0;
+    border: 2px solid #000;
+    background: #FFFFFF;
+    border-radius: 5px;
+}
+.new_data{
+    border: none;
+    border-bottom: 1px solid #000;
+    border-radius: 0;
+    padding-left: 0;
+    padding-right: 0;
+}
+
+.fr-toolbar.fr-top{
+    /* display: none; */
```

Comment:
@RutaleIvanPaul, Why does this CSS class empty?

Reply:
It has commented out code that you are not using, so you could just delete the class

Reply:
I have changed this.